docs/docs/guides/initializing-schemas.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Initializing your Schema
+id: initializing-your-schema
+hide_title: false
+slug: /guides/initializing-schemas
+---
+
+## Background
+
+Neosync has a feature that will "Initialize" your schema. However this feature is not always the best way to get your schema set up. This guide will walk you through ways to set up your schema manually.
+
+## Option 1: Run your Migrations
+
+If you have a set of migrations that you use to set up your database, you can run these migrations on your target database. This is the most common way to set up your schema.
+
+This can potentially have some issues if you have a lot of migrations or if your migrations are not idempotent.
+
+If your migrations are any of the following you may run into issues:
+
+- Not Idempotent, meaning that running the migration multiple times (or against a completely blank/fresh database) will cause issues
+- Have hard coded database names (if you are choosing to run neosync against a separate logical database in your single instance)
+- Edit database users, global configurations or other settings that are not related to the schema/data
+
+Some of the above issues can be resolved by choosing to use a completely fresh database, or by running your migrations against a fresh database and then snapshotting that database to use as your target.
```

Comment:
snapshotting seems like a valid word. 
You can exclude it for this file by adding an exclusion rule to the metadata.

```
# cSpell:words snapshotting
```

Reply:
This sentence didn't make a ton of sense as I was reading it back so reworded it slightly